Make.inc

```diff
@@ -132,6 +132,10 @@ else
 JULIA_COMMIT := $(JULIA_VERSION)
 endif
 
+# Increase these every release when breaking ABI in a backward-incompatible way
```

Comment:
A solution I find natural, and which works well with semantic versioning, it to have SOMAJOR equal to the major Julia version, and SOMINOR to the minor version. The parallel between SOVERSION and the official version makes things clearer.

Until 1.0, SOMAJOR would be equal to `0.$minorversion` and SOMINOR to `$patchversion`.